package.json

```diff
@@ -198,12 +269,10 @@
             "razor"
           ]
         },
-        "runtime": "node",
```

Comment:
The changes from here down need to be undone as this breaks the debugger proxy work. the debugger proxy allows us to run the debugger in the same session that the extension is installed. We do this by rewriting the manifest on installation, so unfortunately this will change every time you f5 during development of the extension.